src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/FixView.tsx

```diff
@@ -96,6 +97,15 @@ export const FixView = (props: FixViewProps) => {
     );
   };
 
+  const currentStepIndex = stepLinks.findIndex((stepLink) =>
```

Comment:
I figured I'd just let the previous and next arrows link to the previous and next step in the flow, but that doesn't quite work well, because I can't determine the current step based on the URL (the Scan step has multiple URLs, for example, depending on if the scan already ran).

Thus, I think I'll do a followup PR where I'll remove this component from the layout, and move it into every individual step, and just make the next step use the same `getRelevantGuidedSteps` function.

Reply:
Yeah at least those, but I'm also weary of potential other pages where this is the case - either current ones, that I'm not aware of, or future ones. e.g. the celebration steps that we're eventually/already adding? Hence the followup PR to re-use the step determination logic.